torch/csrc/Stream.cpp

```diff
@@ -43,6 +43,22 @@ static PyObject* THPStream_pynew(
   END_HANDLE_TH_ERRORS
 }
 
+PyObject* THPStream_New(const c10::Stream& stream) {
```

Comment:
The `new` name is usually reserved for the tp_new slow function. We usually use `wrap()` for the converter from c++ object to python object. Can you rename this?